roles/etcd_migrate/tasks/migrate.yml

```diff
@@ -42,6 +44,7 @@
     --lease-duration 1h
   environment:
     ETCDCTL_API: 3
+    PATH: "/usr/local/bin:/var/usrlocal/bin:{{ ansible_env.PATH }}"
```

Comment:
It is set to `/usr/local/bin/oadm` if the deployment is containerized. Based on `grep -rn admin_binary` I was not able to find any assignment for the AH. Plus, the fact does not cover `/var/usrlocal/bin` case.